.github/workflows/release-dev.yml

```diff
@@ -0,0 +1,52 @@
+name: Publish Development App
+
+on:
+  release:
+    types: [created]
```

Comment:
Should this not be on all merges to the branch instead?

Reply:
Thanks for catching this